### PR TITLE
add redirections for go.k8s.io

### PR DESCRIPTION
**Pull request by nitishfy**
Fixes: #6242

### CODE REVIEW

File: apps/k8s-io/README.md [hunk 1]
```diff
@@ -57,6 +57,8 @@ Redirections
 - https://go.k8s.io/test-history
 - https://go.k8s.io/triage
 - https://go.k8s.io/logo
+- https://go.k8s.io/contact
+- https://go.k8s.io/contact/${GROUP_NAME}
```
Comment (MadhavJivrajani) [inline apps/k8s-io/README.md:61]: Let's also add an example here as a sub-bullet point so that folks know what this redirects to
```suggestion
- https://go.k8s.io/contact/${GROUP_NAME}
  - For example: 
    - https://go.k8s.io/contact/sig-contributor-experience
    - https://go.k8s.io/contact/wg-lts
    - https://go.k8s.io/contact/committee-steering
```

File: apps/k8s-io/configmap-nginx.yaml [hunk 2]
```diff
@@ -306,7 +306,9 @@ data:
           rewrite ^/stuck-prs$       https://github.com/kubernetes/kubernetes/pulls?utf8=%E2%9C%93&q=is%3Apr%20is%3Aopen%20label%3Algtm%20label%3Aapproved%20-label%3Ado-not-merge%20-label%3Aneeds-rebase%20sort%3Aupdated-asc%20-status%3Asuccess redirect;
           rewrite ^/test-history$    https://storage.googleapis.com/kubernetes-test-history/static/index.html redirect;
           rewrite ^/triage$          https://storage.googleapis.com/k8s-triage/index.html redirect;
-          rewrite ^/logo$            https://branding.cncf.io/projects/kubernetes/ redirect;
+          rewrite ^/logo$            https://branding.cncf.io/tree/main/projects/kubernetes redirect;
+          rewrite ^/contact$         https://github.com/kubernetes/community/tree/master/communication/ redirect;
+          rewrite ^/contact/(.*)$     https://github.com/kubernetes/community/tree/master/$1#contact redirect;
```
[the remark this answers is not in the document]
Comment (MadhavJivrajani) [reply apps/k8s-io/configmap-nginx.yaml:311]:
```suggestion
          rewrite ^/contact/([^/]*)/?$ https://github.com/kubernetes/community/tree/master/$1#contact redirect; 
```
So that it is consistent with other such similar redirects ^

Please align the line as you see best ^

File: apps/k8s-io/configmap-nginx.yaml [hunk 3]
```diff
@@ -306,7 +306,9 @@ data:
           rewrite ^/stuck-prs$       https://github.com/kubernetes/kubernetes/pulls?utf8=%E2%9C%93&q=is%3Apr%20is%3Aopen%20label%3Algtm%20label%3Aapproved%20-label%3Ado-not-merge%20-label%3Aneeds-rebase%20sort%3Aupdated-asc%20-status%3Asuccess redirect;
           rewrite ^/test-history$    https://storage.googleapis.com/kubernetes-test-history/static/index.html redirect;
           rewrite ^/triage$          https://storage.googleapis.com/k8s-triage/index.html redirect;
-          rewrite ^/logo$            https://branding.cncf.io/projects/kubernetes/ redirect;
+          rewrite ^/logo$            https://branding.cncf.io/tree/main/projects/kubernetes redirect;
```
Comment (MadhavJivrajani) [inline apps/k8s-io/configmap-nginx.yaml:309]: Can you pls split this out into a separate commit?

File: apps/k8s-io/test.py [hunk 4]
```diff
@@ -212,6 +211,13 @@ def test_go(self):
             self.assert_temp_redirect(
                 base + 'needs-ok-to-test',
                 'https://github.com/pulls?q=org%3Akubernetes+org%3Akubernetes-sigs+org%3Akubernetes-csi+org%3Akubernetes-client+is%3Aopen+is%3Apr+label%3Aneeds-ok-to-test+label%3A%22cncf-cla%3A+yes%22+-label%3Aneeds-rebase')
+            self.assert_temp_redirect(
+                base + 'contact',
+                'https://github.com/kubernetes/community/tree/master/communication')
```
Comment (MadhavJivrajani) [inline apps/k8s-io/test.py:216]:
```suggestion
                'https://github.com/kubernetes/community/tree/master/communication/')
```
nit: just so that it is consistent with what we have in our nginx config.

**Review by MadhavJivrajani — CHANGES_REQUESTED** [review]
Thank you for working on this @nitishfy ❤️ 

Just a few minor comments.

File: apps/k8s-io/configmap-nginx.yaml (hunk 3 above)
Comment (MadhavJivrajani) [reply apps/k8s-io/configmap-nginx.yaml:309]: Actually, let's split this off into another PR
We will need to change the redirect here, in the test file and in the README.

**Review by MadhavJivrajani — APPROVED** [review]

File: apps/k8s-io/configmap-nginx.yaml [hunk 5]
```diff
@@ -306,7 +306,8 @@ data:
           rewrite ^/stuck-prs$       https://github.com/kubernetes/kubernetes/pulls?utf8=%E2%9C%93&q=is%3Apr%20is%3Aopen%20label%3Algtm%20label%3Aapproved%20-label%3Ado-not-merge%20-label%3Aneeds-rebase%20sort%3Aupdated-asc%20-status%3Asuccess redirect;
           rewrite ^/test-history$    https://storage.googleapis.com/kubernetes-test-history/static/index.html redirect;
           rewrite ^/triage$          https://storage.googleapis.com/k8s-triage/index.html redirect;
-          rewrite ^/logo$            https://branding.cncf.io/projects/kubernetes/ redirect;
+          rewrite ^/logo$            https://branding.cncf.io/projects/kubernetes/ redirect;          rewrite ^/contact$         https://github.com/kubernetes/community/tree/master/communication/ redirect;
+          rewrite ^/contact/([^/]*)/?$ https://github.com/kubernetes/community/tree/master/$1#contact redirect;
```
Comment (MadhavJivrajani) [inline apps/k8s-io/configmap-nginx.yaml:310]: I think you removed the wrong line?
`/contact` is now missing

**Review by MadhavJivrajani — CHANGES_REQUESTED** [review]
/approve cancel
/lgtm cancel

File: apps/k8s-io/configmap-nginx.yaml (hunk 5 above)
Comment (nitishfy) [reply apps/k8s-io/configmap-nginx.yaml]: resolved

**Comment by nitishfy** [comment]
> @nitishfy please squash commits and this will be good to go /hold

Done!

**Review by MadhavJivrajani — APPROVED** [review]
